<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetAddFunction</class>
 <widget class="QWidget" name="WidgetAddFunction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加函数</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>名称：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit_name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_description">
     <property name="text">
      <string>描述：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEdit_description"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_parameters">
     <property name="text">
      <string>参数：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_parameters">
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_expression">
     <property name="text">
      <string>函数体：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_expression"/>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_add_parameter">
     <property name="text">
      <string>添加参数</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pushButton_add_function">
     <property name="text">
      <string>添加函数</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
